Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -119,9 +119,6 @@ type BocAutoCompleteReferenceValue_PositioningOptions = {
 
 interface JQueryStatic {
```

Comment:
I think we should replace the jquery type with a custom type that only exposes the apis we still "support" at this time. Unless that's really a lot of work instead of just an hour or so.

Reply:
The jquery type is removed in the follow up issue. Most likely, it is going to be replaced by a class that inherits from an HTML element. So this should resolve itself.

Reply:
My idea was, to make sure with typesafety that we catch all cleanups that we want to do here. What process did you use to ensure that we don't have a jquery.getouterwidth() or jquery.data() or some other jquery function not releated to wiring-up or events left in the code?

Reply:
I have removed all mentions of `$` and `jQuery` from the file, going through each and evaluating if it can be replaced. The only jQuery left are the arguments passed into the autocomplete function and the jQuery construct itself. To check I also removed the jQuery types and checked each of the ocurring errors since those are related to jQuery.